docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -373,6 +373,11 @@ statements, the connector supports the following features:
 - {doc}`/sql/show-create-table`
 - {doc}`/sql/comment`
 
+### ALTER TABLE
+
+The connector supports the `ALTER TABLE RENAME TO` command only when used with a
+Glue metastore.
```

Comment:
We can rename **_external_** tables with Thrift metastore. Can we rephrase this sentence?

Reply:
> The connector supports the `ALTER TABLE RENAME TO`  when the operation does not involve moving the content of the table to a new location on the object storage.

Concrete examples where renaming a table is supported:
- the table type is _external_
- the table is backed by a metastore which does not perform object storage operations, e.g AWS Glue